### ISSUE TITLE
Requesting a feature to blacklist specific pokemob spawns near a defined structure

### ISSUE THREAD

**Issue by lunapanshiel** [Pokecube-Development/Pokecube-Issues-and-Wiki#906]
As the title suggests, I would very much like the ability to block a specific pokemob from spawning if it happens to be within X blocks of certain structures

For example, in my current game which I have been using to test and tweak configs, I have multiple mods that add new kinds of structures/dungeons, a persistent problem is because many of them are underground, geodudes and gravelers spawn in and around them, and are prone to exploding with slight provocation. This is easily a big problem in underground cities that can accumulate huge craters in a short amount of time

I'd like the ability to have, for example: mobblacklist ("pokecube:village" : "graveler" : "24") where the listed pokemob is NOT allowed to spawn within 24 blocks of any village structures

for now I have disabled block damage for explosions(which seems overly high by default, none of the exploded blocks actually seem to drop as items at all) but I'd love to just keep the griefy pokemobs away from a location where NPCs can blow up chunks of their town

Alternately, and probably easier: have a "mark structure as "damp"" setting, so exploding pokemob shenanigans are completely disallowed in the vicinity. It would be pretty easy to argue most abandoned structures and ruins are damp anyway

**Comment by Thutmose** [comment]
Hmm, I can look into options for this, it does seem like a generally good idea. This can already be done to prevent them spawning "inside" the structure, but that is it for now

**Comment by lunapanshiel** [comment]
> Hmm, I can look into options for this, it does seem like a generally good idea. This can already be done to prevent them spawning "inside" the structure, but that is it for now

yeah i already went to the effort to do that, though they still keep coming in from the surrounding biomes which is a problem. especially bad in underground towns within pokecube itself
